guides/advanced-use/asset-compilation.md

```diff
@@ -53,7 +53,7 @@ module.exports = {
 
 <!-- alex disable simply -->
 If you need more fine-grained customization over the way that `babel-preset-env` transforms your code,
-simply set any of the options found [here](https://github.com/babel/babel-preset-env#options) on your application's `babel` hash in `ember-cli-build.js`.
+simply set any of the options found [here](https://babeljs.io/docs/en/babel-preset-env.html#options) on your application's `babel` hash in `ember-cli-build.js`.
```

Comment:
Rather than using the word `here` as the link description, let's provide a descriptive information for accessibility.

```suggestion
simply set [options for the preset](https://babeljs.io/docs/en/babel-preset-env.html#options) on your application's `babel` hash in `ember-cli-build.js`.
```

Reply:
Good call, updated the link text